05-networks/user/user.ts

```diff
@@ -0,0 +1,122 @@
+import { IncomingMessage, ServerResponse } from "http";
+import { USERS, User } from "./users.const";
+import { IncomingMessageWithBody } from "../models/inccoming-message-with-body.interface";
+import * as http from "http";
+
+const createUser = (req: IncomingMessageWithBody, res: ServerResponse) => {
+  console.log("create");
```

Comment:
In production code you want to remove console.logs unless it is part of the logging logic (log an error for example) when usually a logging library is used.